q.js

```diff
@@ -1645,9 +1645,12 @@ function any(promises) {
         function onRejected(err) {
             pendingCount--;
             if (pendingCount === 0) {
-                err.message = ("Q can't get fulfillment value from any promise, all " +
+                var rejection = err || new Error("Rejection was null/undefined");
```

Comment:
This could be new Error("error " + err) and you’d get "error undefined" or "error null" specifically.